### DISCUSSION TITLE
Étoffer le missionsshop

### DISCUSSION THREAD

**Discussion by KirIcare** [Crownicles/Crownicles#1685] [Nouvelles idées]
Comme son nom l'indique, ça consisterait à ajouter de nouvelles options disponibles dans le missionsshop.
On a pensé à un "conseils du maître-d'armes de la cour royale" qui une fois acheter avec des gemmes (évidemment) donnerais de l'xp
On pourrais soit le faire de manière fixe ou alors varier la valeur de l'xp comme avec l'argent du roi.
Le tout avec un cooldown d'une semaine comme pour les pts afin d'éviter les abus.
Une petite souris m'a également soufflé la bonne idée de faire un choix, si je prends l'xp, je ne pourrais pas acheter les pts pendant la même semaine et inversement.

Sur les mêmes principes on pourrais également faire un "conseils de Katrin" mais avec de l'xp de guilde. Évidemment on peut changer le nom, j'ai juste mis ça en réf a la guilde des aventuriers dans genshin impact.

**Comment by romain22222** [comment]
> Comme son nom l'indique, ça consisterait à ajouter de nouvelles options disponibles dans le missionsshop. On a pensé à un "conseils du maître-d'armes de la cour royale" qui une fois acheter avec des gemmes (évidemment) donnerais de l'xp On pourrais soit le faire de manière fixe ou alors varier la valeur de l'xp comme avec l'argent du roi. Le tout avec un cooldown d'une semaine comme pour les pts afin d'éviter les abus. Une petite souris m'a également soufflé la bonne idée de faire un choix, si je prends l'xp, je ne pourrais pas acheter les pts pendant la même semaine et inversement.
> 
> Sur les mêmes principes on pourrais également faire un "conseils de Katrin" mais avec de l'xp de guilde. Évidemment on peut changer le nom, j'ai juste mis ça en réf a la guilde des aventuriers dans genshin impact.

moi je serais partant, mais je sais que draft ne le sera pas pour la raison suivante : les gemmes ne doivent pas servir à acheter une autre ressource, ou alors ca doit être à un taux dérisoire (à la base les pts, c'était pour se moquer des gens qui montent dans le classement)

**Reply by KirIcare** [reply]
Une autre ressource... L'argent n'en ai pas une ?

**Reply by romain22222** [reply]
as tu déjà acheté l'argent du roi ?

**Reply by romain22222** [reply]
hormis pour la quête bien sur ?

**Reply by Pietagorh** [reply]
Oui, 1) c'est la mission 51
2) ça a été grandement buff

**Reply by KirIcare** [reply]
Savoir si je l'achète ou pas n'est pas la question, l'option existe. Rien ne dit qu'il n'y aura pas une personne dans draftbot qui n'achetera jamais l'xp
Enfin bref content que ça te plaise et dommage si draft n'en veux pas, c'est pas la première suggestion qui passe à la trappe et si personne ne suggère, comment savoir si c'est OK ou non ? X)

**Comment by BastLast** [comment]
je suis plus interessés par des moyens de rendre le vétérinaire interessant plutôt que des moyens d'acheter de l'xp avec des gemmes

**Reply by Pietagorh** [reply]
Permets au passe mission de faire passer des missions de campagne et on en parlera 👀

**Reply by BastLast** [reply]
la campagne a pour objectif d'être faite entièrement, passer des missions de la campagne serait complètement anti-jeu

**Reply by KirIcare** [reply]
Bah... Puisque c'est un vétérinaire, il fait le check up classique comme maintenant et puis il donne un petit remontant à ton pet et il gagne 8~9 pts d'amour?

**Reply by BastLast** [reply]
Go en discuter dans la discussion dédiée tho
